src/components/ScholarshipForm.jsx

```diff
@@ -239,59 +242,97 @@ function ScholarshipForm({ scholarship }) {
   }
 
   const onLastStep = activeStep == Object.keys(stepperItems).length - 1;
+  const lintIssues =
+    activeStep === 1 ? lint(formik.values) : { issues: {}, lintVals: {} };
```

Comment:
had to do this to avoid an error when using Object.keys() below

Reply:
You can avoid this by embedding the alert within Requirements step. Between these two lines would be a good place I think:
<img width="452" alt="Screen Shot 2022-02-22 at 12 03 50 AM" src="https://user-images.githubusercontent.com/8023233/155088627-463ce87b-1b6f-401a-9504-c7601c75e454.png">